integration/monai/monai_nvflare/client_algo_executor.py

```diff
@@ -104,6 +109,11 @@ def execute(self, task_name: str, shareable: Shareable, fl_ctx: FLContext, abort
         self.log_info(fl_ctx, f"Client trainer got task: {task_name}")
 
         try:
+            self.log_warning(fl_ctx, f"EXECUTING task: {task_name}")
+            import resource
+
+            rlimit = resource.getrlimit(resource.RLIMIT_NOFILE)
+            resource.setrlimit(resource.RLIMIT_NOFILE, (4096, rlimit[1]))
```

Comment:
Why do we need these lines for?

Reply:
Did not mean to include this file in the commit, I had to do this to fix an error I was running into locally